File: Lista1/Python/ex_9.py
numero = int(input('Digite um numero para saber seu fatorial: '))
print(f'Fatorial de {numero}')
print(f'{numero}! = ',end='')
fatorial = numero
for c in range(numero, 0, -1):
    if c < numero:
        fatorial *= c
print(fatorial)

File: Lista1/Java/ex_9.java
import java.util.Scanner;
public class ex_9
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
        String expressao = "";
        int fatorial = 1;
        System.out.println("Digite um número para saber seu fatorial: ");
        int numero = sc.nextInt();
        System.out.println("Fatorial do número " + numero);
        for (int c = numero; c >= 1; c--) {
            fatorial *= c;
            expressao += c;
            if (c > 1) {
                expressao += " x ";
            }
        }
        System.out.println(numero + "! = " + expressao + " = " + fatorial);
    }
}